src/pages/downloads/server.tsx

```diff
@@ -62,6 +62,12 @@ export default function DownloadsPage({ osType = OsType.Linux }: { osType?: OsTy
                 >
                   macOS
                 </Link>
+                <Link
+                  to='/downloads/dotnet'
+                  className={clsx('pills__item', { 'pills__item--active': osType === OsType.DotNet })}
+                >
+                  DotNet
```

Comment:
```suggestion
  .NET Core
```

Reply:
It should be .NET as .NET Core no longer exists after 6.0